config/manager/manager.yaml

```diff
@@ -22,8 +22,6 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
-      securityContext:
-        runAsUser: 65532
```

Comment:
I know that this is done to make it work with OCP but from a k8s point of view this is very very good to have. Personally I would like to lock it down even more by default.

In short I would prefer not to remove it.
Overall I have been thinking on how we should make our yaml available to our users and since we don't support any helm chart I guess kustomize is the natural step especially since kubebuilder/operator-sdk uses it so much. At least for us none OLM users.

We could draw some experience from the fluxcd team and they seem to be using a simple kustomizeation for install.
https://github.com/fluxcd/flux2/blob/main/manifests/install/kustomization.yaml

We could have a install/base that we suggest for k8s users.
And we can have a install/ocp that calls on base but with a patch that removes securityContext.

But I do not know how that would work together with OLM.

What do you think?

Reply:
We have solved this in a separate kustomize file and update the documentation to match.